spec.html

```diff
@@ -36114,6 +36247,21 @@ <h1>get RegExp.prototype.global</h1>
         </emu-alg>
       </emu-clause>
 
+      <emu-clause id="sec-get-regexp.prototype.hasIndices">
+        <h1>get RegExp.prototype.hasIndices</h1>
+        <p>`RegExp.prototype.hasIndices` is an accessor property whose set accessor function is *undefined*. Its get accessor function performs the following steps:</p>
+        <emu-alg>
+          1. Let _R_ be the *this* value.
+          1. If Type(_R_) is not Object, throw a *TypeError* exception.
+          1. If _R_ does not have an [[OriginalFlags]] internal slot, then
```

Comment:
I think this now needs to call into RegExpHasFlag, like the other accessors have been updated to do?